test/test-applications/integrations/Samples.Owin.WebApi2/Controllers/HomeController.cs

```diff
@@ -11,5 +12,26 @@ public string IsAlive()
         {
             return "Yes";
         }
+
+        [HttpGet]
+        [Route("shutdown")]
+        public string Shutdown()
+        {
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+
+            foreach (var assembly in assemblies)
+            {
+                foreach (var type in assembly.DefinedTypes)
+                {
+                    if (type.Namespace == "Coverlet.Core.Instrumentation.Tracker")
+                    {
+                        var unloadModuleMethod = type.GetMethod("UnloadModule", System.Reflection.BindingFlags.Public | System.Reflection.BindingFlags.Static);
+                        unloadModuleMethod.Invoke(null, new object[] { this, EventArgs.Empty });
+                    }
+                }
+            }
```

Comment:
This code and `IsProfilerAttached()` are repeated so many times, make we want to extract them into a shared assembly someday. (Sorry, I can't help myself.)

Reply:
Probably not a shared assembly because some test projects need to be careful about their dependencies. But we can use a shared project to avoid code duplication.

Reply:
Good idea. I always forget that shared projects exist.